docs/content/manual/pre-release/node/degraded-availability.md

```diff
@@ -1,22 +1,32 @@
 ---
 title: Degraded availability with added nodes
 ---
-https://github.com/longhorn/longhorn/issues/1701
 
 #### Volume creation using UI with degraded availability and added node
-Start with one node cluster.
 
+##### Related Issue:
+- https://github.com/longhorn/longhorn/issues/1701
+
+##### Prerequisites:
+- Start with 1 node cluster.
+- Double check if "Allow Volume Creation with Degraded Availability" is ticked or return **true** with following command:
+    - `kubectl get settings.longhorn.io/allow-volume-creation-with-degraded-availability -n longhorn-system`
+
+##### Steps:
 1. Create a Deployment Pod with a volume and three replicas.
-    1. After the volume is attached, `scheduling error` should be seen from the UI. But no functional impact.
+    1. After the volume is attached, on Volume page it should be displayed as `Degraded`
+    1. Hover the crusor to the red circle exclamation mark, the tooltip will says, "The volume cannot be scheduled".
+    1. Click into the volume detail page it will display `Scheduling Failure` but the volume remain fuctional as expected.
```

Comment:
just NIT, all steps are 1?

Reply:
I believe this is lazy numbering from markdown standard, so you don't need to keep counting.

Reply:
ok